README.rst

```diff
@@ -134,9 +136,9 @@ within the specified script, **in addition to running every command** in the scr
 explain output is generated using the `logging <https://docs.python.org/3/library/logging.html>`_ module,
 if your script configures logging module there are certain things to keep in mind:
 
-- if your script sets the `logging level <https://docs.python.org/3/library/logging.html#logging-levels>`_
-    higher than INFO, the explain output will be suppressed entirely.
-- the explain output will be sent to whatever stream your script configures the logging module to send output to.
+    - if your script sets the `logging level <https://docs.python.org/3/library/logging.html#logging-levels>`_
+        higher than INFO, the explain output will be suppressed entirely.
+    - the explain output will be sent to whatever stream your script configures the logging module to send output to.
```

Comment:
This still doesn't look correct in the rendered view: https://github.com/mongodb-labs/pymongoexplain/blob/b4bd835c930b496f0641a6a81f511244a21c179e/README.rst#explaining-commands-in-a-script

Reply:
Done.